### PR TITLE
FIX Read the Docs build.

### PR DESCRIPTION
**Pull request by moromimay**
## Why? :open_book:
Read the Docs was unable to build the documentation because of the h3 version. I took the opportunity to update the documentation as well.

## What? :wrench:
- README
- Docs
- Requirements

## Type of change
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How everything was tested? :straight_ruler:
You can see the build [here](https://readthedocs.org/projects/butterfree/builds/12473033/)

## Checklist
- [x] I have made corresponding changes to the documentation;

### CODE REVIEW

**Review by fvcaputo — DISMISSED** [review]
Nice :ok_hand:

File: requirements.dev.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-cmake==3.13.3
-h3==3.4.3
+cmake==3.18.4.post1
```
Comment (AlvaroMarquesAndrade) [inline requirements.dev.txt:1]: Maybe we could go with version ```3.18.4```, I don't think it's necessary for us to use this post release. What do you think?

**Review by AlvaroMarquesAndrade — COMMENTED** [review]
Just a small suggestion, but LGTM overall.

File: requirements.dev.txt (hunk 1 above)
Comment (moromimay) [reply requirements.dev.txt:1]: I will test this version :smile:

**Review by AlvaroMarquesAndrade — APPROVED** [review]

**Pull request merged**